lucida/botframework-interface/interface.js

```diff
@@ -0,0 +1,91 @@
+start_time = process.hrtime()
+var time_offset = Date.now() / 1000 - start_time[0] - start_time[1] / 1000000000;
+delete start_time;
+
+function time() {
+  var hr_time = process.hrtime()
+  return time_offset + hr_time[0] + hr_time[1] / 1000000000;
+}
+
```

Comment:
This too was removed in [20d80ac](https://github.com/claritylab/lucida/pull/175/commits/20d80aca0ef26674a12ab5e8698fd40e0ded8abf).